README.md

```diff
@@ -3403,6 +3403,16 @@ Other Style Guides
     - We primarily use [`mocha`](https://www.npmjs.com/package/mocha) at Airbnb. [`tape`](https://www.npmjs.com/package/tape) is also used occasionally for small, separate modules.
     - 100% test coverage is a good goal to strive for, even if it’s not always practical to reach it.
     - Whenever you fix a bug, _write a regression test_. A bug fixed without a regression test is almost certainly going to break again in the future.
+    
+  <a name="testing--is-documentation"></a>
+  - [30.3](#testing--is-documentation) **Is your documentation**:
+    - Tests should explain what your code does.
+    - First write a case for each main functionality, and then their exceptions.
+    - Refactor tests until they are easy to read and understand.
+    - Never use _"private"_ properties to test code.
```

Comment:
i'm not sure this really belongs; if something is truly private, it's impossible to test directly; and if it's possible to test directly, it's not actually private.

Reply:
Uhmmm, I understand your point, I guess that it comes from here:
- https://github.com/airbnb/javascript#naming--leading-underscore
> This convention might lead developers to wrongly think that a change won’t count as breaking, or that tests aren’t needed.

What about changing it for the following statement:

    - JavaScript does not have the concept of privacy in terms of properties or methods,
      if you want something to be “private”, it must not be observably present,
     and they won't be used in tests, see [here](#naming--leading-underscore) for details.

Reply:
I think this is kind of self-evident; and would be more clear with a concrete example of what to avoid.